parent/pom.xml

```diff
@@ -1870,6 +1870,18 @@
                 <version>${cxf.opentracing.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>io.opentelemetry</groupId>
+                <artifactId>opentelemetry-bom-alpha</artifactId>
```

Comment:
Why do we need opentelemetry-bom-alpha and not opentelemetry-bom?

Reply:
The only reason but not the least is to have the version of opentelemetry-semconv handled by this BOM. This module has static variables with the standard attribute names from the OpenTelemetry semantics conventions. They are however not "stable" yet, hence the alpha version.

Reply:
If this is just one single artifact, it would be better to have `opentelemetry-semconv` defined separately (like `opentelemetry-opentracing-shim`), it is a bit risky to make a releases with alpha artifacts (that are not test related).

Reply:
i switch for opentelemetry-bom with semconv on the side, and introduced maven properties for versioning